src/storage/data.rs

```diff
@@ -114,18 +114,18 @@ impl<'a, T> SystemData<'a> for ReadStorage<'a, T>
 where
     T: Component,
 {
-    fn fetch(res: &'a Resources, id: usize) -> Self {
-        Storage::new(res.fetch(0), res.fetch(id))
+    fn fetch(res: &'a Resources) -> Self {
+        Storage::new(res.fetch(), res.fetch())
```

Comment:
This is kinda unreadable if you don't know which types of resources these fetch (But the `reads` method does tell the types..).